Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -28,6 +28,20 @@ async function updateKuduDeploymentLog(endPoint, webAppName, resourceGroupName,
     }
 }
 
+async function waitForAppServiceToStart(endPoint, resourceGroupName, webAppName, specifySlotFlag, slotName) {
+    var appServiceDetails = await azureRmUtil.getAppServiceDetails(endPoint, resourceGroupName, webAppName, specifySlotFlag, slotName);
+    if(appServiceDetails.hasOwnProperty("properties") && appServiceDetails.properties.hasOwnProperty("state"))
+    {
+        tl.debug('App Service State : ' + appServiceDetails.properties.state);
+        while(!(appServiceDetails.properties.state == "Running" || appServiceDetails.properties.state == "running"))
+        {
+            appServiceDetails = await azureRmUtil.getAppServiceDetails(endPoint, resourceGroupName, webAppName, specifySlotFlag, slotName);
```

Comment:
We should  check for hasProperty("properties") and hasProperty("state") after getting appServiceDetails in while loop